src/functor.jl

```diff
@@ -48,6 +48,24 @@ function params!(p::Params, x, seen = IdSet())
   end
 end
 
+"""
+    params(model)
```

Comment:
```suggestion
    params(model)
    params(layers...)
```
Having just the one vararg signature might be simpler, but given 90% of people would be passing in a single model struct my worry would be that it would be confusing.

Reply:
this makes sense to me